src/main/java/com/authlete/jaxrs/BaseResourceEndpoint.java

```diff
@@ -95,14 +95,14 @@ public String extractAccessToken(String authorization, String accessTokenInReque
      */
     public AccessTokenInfo validateAccessToken(AuthleteApi api, String accessToken) throws WebApplicationException
     {
-        return validateAccessToken(api, accessToken, null, null);
+        return validateAccessToken(api, accessToken, null, null, null);
     }
 
 
     /**
      * Validate an access token. This method is an alias of {@link
      * #validateAccessToken(AuthleteApi, String, String[], String)
-     * validate}<code>(api, accessToken, requiredScopes, null)</code>.
+     * validate}<code>(api, accessToken, requiredScopes, null, null)</code>.
```

Comment:
Sorry, the JavaDoc before the change was wrong.
`validate}` should be changed to `validateAccessToken}`